CSSSSupportLibrary/SupportLibrary/OSVersionInfo/WinNTVersionInfo.OSVersionInfo.SupportLibrary.cs

```diff
@@ -272,6 +274,9 @@ static public string Edition
                     #region VERSION 6
                     else if (majorVersion == 6)
                     {
+                        // Since VS 2015 is a build image, the use of "out var x" cannot
+                        // be used. If VS 2015 support is dropped, then the use of this
+                        // can be implemented as it is part of C# 7.0

```

Comment:
Can this be changed (to make it clear why VS 2015 is a build image):

```
// Since VS 2015 is a build image on AppVeyor (for Windows Server 2012 R2
// support) the use of `out var x` cannot be used. Once support for this has
// been dropped, the use of this can be implemented as it is part of C# 7.0
```

(Comment line length can be changed as needed)